streams/src/main/java/org/apache/kafka/streams/kstream/aggregates/CountLong.java

```diff
@@ -15,19 +15,18 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.test;
+package org.apache.kafka.streams.kstream.aggregates;
 
-import org.apache.kafka.streams.kstream.Initializer;
+import org.apache.kafka.streams.kstream.Aggregator;
 
-public class MockInitializer {
 
-    private static class StringInit implements Initializer<String> {
-
-        @Override
-        public String apply() {
-            return "0";
-        }
+public class CountLong<K, V> implements Aggregator<K, V, Long> {
```

Comment:
Since `CountLong` doesn't care about the key and value, we should be able to remove these type parameters.